pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -2024,6 +2026,35 @@ var _ = Describe("Converter", func() {
 			Entry("use virtio transitional", true),
 			Entry("use virtio non-transitional", false),
 		)
+		DescribeTable("Should set the error policy", func(epolicy *v1.DiskErrorPolicy, expected string) {
+			vmi.Spec.Domain.Devices.Disks[0] = v1.Disk{
+				Name: "mydisk",
+				DiskDevice: v1.DiskDevice{
+					Disk: &v1.DiskTarget{
+						Bus: v1.VirtIO,
+					},
+				},
+			}
+			vmi.Spec.Domain.Devices.Disks[0].ErrorPolicy = epolicy
+			vmi.Spec.Volumes = append(vmi.Spec.Volumes, v1.Volume{
```

Comment:
Any reason why `vmi.Spec.Domain.Devices.Disks[0]` is overwritten while for `vmi.Spec.Volumes` the `append` call is used?

Reply:
Yes, you are right. They should be consistent. I opted for indexing it as it is a bit easier to check the result with Expect